python/foxglove-sdk/python/examples/ws-demo.py

```diff
@@ -29,15 +30,15 @@ def main() -> None:
 
     server = foxglove.start_server(port=8765)
 
+    # Log messages having well-known Foxglove schemas using the appropriate channel type.
+    box_chan = SceneUpdateChannel("/boxes")
+    tf_chan = FrameTransformChannel("/tf")
+
+    # Log JSON messages using the JsonEncoder.
+    # A ProtobufEncoder is also available.
     sin_chan = foxglove.Channel(
         topic="/sine", encoder=foxglove.JsonEncoder(), schema=plot_schema
     )
-    box_chan = foxglove.Channel(
-        topic="/boxes", encoder=foxglove.ProtobufEncoder(), schema=SceneUpdate
-    )
```

Comment:
did we lose the 3d examples? plan to bring them back?

Reply:
Still here. I moved the Foxglove schema examples to the top; this one now looks like:

```py
box_chan = SceneUpdateChannel("/boxes")
```